testutils/build.gradle

```diff
@@ -198,6 +198,7 @@ dependencies {
         implementation(group: 'com.microsoft.identity', name: 'labapi', version: '0.0.+')
     }
 
+    implementation "org.mockito:mockito-inline:$rootProject.ext.mockitoCoreVersion"
```

Comment:
Once we address this: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1472/files#r671562680 then probably don't need this depedency

Reply:
Do we need this Mockito depedency? Not clear if this is being used somewhere in this PR

Reply:
```suggestion
```